docs/reference/how-to/knn-search.asciidoc

```diff
@@ -17,7 +17,9 @@ The default <<dense-vector-element-type,`element_type`>> is `float`. But this
 can be automatically quantized during index time through
 <<dense-vector-quantization,`quantization`>>. Quantization will reduce the
 required memory by 4x, but it will also reduce the precision of the vectors and
-increase disk usage for the field (by up to 25%).
+increase disk usage for the field (by up to 25%). Increased disk usage is a
+result of {es} storing both the quantized and the unquantized vectors.
+For example, an increase in disk usage by 10GB can result in a 30GB reduction in required memory.
```

Comment:
```suggestion
For example, when quantizing 40GB of floating point vectors and extra 10GB of data will be stored for the quantized vectors. Totaling in a disk usage of 50GB, but the memory usage for fast search will be reduced to 10GB.
```

Reply:
I am good with that :). As long as we talk about ending totals and starting totals and not just picking `40` and `10` out of thin air.